src/components/MapSelectors/GlobalSelector.style.js

```diff
@@ -70,13 +70,21 @@ export const StyledInputWrapper = styled.div`
 `;
 
 export const StyledInput = styled.input`
-  border-radius: ${props => (props.isOpen ? '4px 4px 0 0' : '4px')};
-  font-size: 14px;
+  font-size: 1rem;
   line-height: 1.5rem;
   border: none;
   width: 100%;
   padding: 0 1rem 0 0;
 
+  // When the font-size of an input element is smaller than 16px, Safari will
```

Comment:
fwiw I think it happens on chrome too?

Reply:
Oh that's good to know, I'm going crazy trying to debug this in Safari (Safari Developer tools keep crashing when I connect to my iPhone and Safari mobile insists on caching the page forever)